sa/storage-authority.go

```diff
@@ -28,6 +27,8 @@ type SQLStorageAuthority struct {
 	log   *blog.AuditLogger
 }
 
+var _ core.StorageAuthority = &SQLStorageAuthority{}
```

Comment:
Oh, sorry, that was when I was considering futzing with the StorageAuthority interface/SQLStorageAuthority API.

Basically, it makes sure a pointer to SQLStorageAuthority adheres the core.StorageAuthority interface. I was getting build errors in other packages instead of sa when I futzed with it and I wanted to make sure I didn't again. It's occasionally useful if an interface needs to churn some but I'll wait until that actually happens to add this. Will remove.